subgit/core.py

```diff
@@ -3,8 +3,10 @@
 # python std lib
 import logging
 import os
+from os.path import basename
```

Comment:
Import order matters, we group all `import ...` first and all `from ...` after it, sort them alphabetically within these 2 groups